modules/protobuf/23.1/presubmit.yml

```
@@ -0,0 +1,24 @@
+matrix:
+  platform: ["centos7", "debian10", "macos", "ubuntu2004", "windows"]
```

```suggestion
  platform: ["debian10", "macos", "ubuntu2004", "windows"]
```
Just drop centos7, it's way too old.

---

Thanks, done.